help/move-to-zulip-cloud.md

```diff
@@ -0,0 +1,100 @@
+
+{!advantages-of-zulip-cloud.md!}
+
+## Process overview
+
+To move your Zulip organization from your self-hosted server to Zulip
+Cloud, you will need to take the following steps, which are described
+in more detail below:
+
+{start_tabs}
+
+1. [Coordinate with Zulip Cloud support](#coordinate-with-zulip-cloud-support).
+
+2. [Upgrade server to `zulip-cloud-current` branch](#upgrade-server).
+
+3. [Export organization data](#export-organization-data).
+
+{end_tabs}
+
+## Coordinate with Zulip Cloud support
+
+Zulip Cloud is updated as new features and other important product
+changes are completed. We highly recommend coordinating the transition
+to Zulip Cloud with our support team so that the process is smooth and
+swift for everyone.
+
+{start_tabs}
+
+1. Review the sections on upgrading your server and exporting your
+   Zulip organization data below, and create an estimated timeline for
+   when you would be able to have your Zulip organization data export
+   ready.
+
+1. [Create a new Zulip Cloud organization](https://zulip.com/new/)
+   with the subdomain you would like for your Zulip organization. Your
+   Zulip Cloud organization will be hosted at `<subdomain>.zulipchat.com`.
+
+1. Email [support@zulip.com](mailto:support@zulip.com) with the URL of
+   your new Zulip Cloud organization, and your estimated timeline for
+   generating the data export.
+
+1. Our support team will confirm if that timeline works, or coordinate
+   an adjusted timeline that works both for your team as well as ours.
+
+{end_tabs}
+
+## Upgrade server
+
+Zulip Cloud runs the `zulip-cloud-current` branch plus some cherry-picked
+changes. This branch is usually delayed by one to two weeks from `main` to
+allow for recent changes to be validated further prior to being deployed
+to customers.
+
+{start_tabs}
+
+1. Check the [version your Zulip server](/help/view-zulip-version#view-zulip-server-and-web-app-version)
+   is running.
+
+1. If you are running a version lower than the latest major release,
+   [follow these instructions](https://zulip.readthedocs.io/en/stable/production/upgrade.html#upgrading-to-a-release)
+   to upgrade to the latest maintenance release.
+
+1. [Follow these instructions](https://zulip.readthedocs.io/en/stable/production/upgrade.html#upgrading-from-a-git-repository)
+   to upgrade to the `zulip-cloud-current` branch.
+
+{end_tabs}
+
+## Export organization data
+
+!!! warn ""
+
+    Before exporting your data, you’ll want to shut down access to the
+    Zulip organization, so that nobody can send new messages (etc.)
+    once you’re exporting data for the Zulip Cloud import.
+
+{start_tabs}
+
+1. [Follow these instructions](https://zulip.readthedocs.io/en/stable/production/export-and-import.html#data-export)
+   to export your Zulip data.
+
+1. Email the resulting data export to [support@zulip.com](mailto:support@zulip.com).
+
+1. Our support team will contact you when the data import for your
+   Zulip Cloud organization is complete.
```

Comment:
I think we might want something here about complications that come with the import process, like the detail about passwords not being included?